### ISSUE TITLE
Imapsync Version 1.998 RFC822.SIZE null!

### ISSUE THREAD

**Issue by samrocks1011** [imapsync/imapsync#271]
Hello 
I am migrating from Icewarp to Zimbra and facing Size Null issue.  As per the issue report I update my imapsync from 1.7 to 1.998 but the issue remains the same.  Please help me out resolve his issue.
OS: CentOS 7

- msg INBOX/437 skipped.
- msg INBOX/438 S[0] F[] I[30-Dec-1899 00:00:00 +0530] has RFC822.SIZE null!
- msg INBOX/438 {0} S[0] F[] I[30-Dec-1899 00:00:00 +0530] could not be fetched:
- msg INBOX/438 skipped.
- msg INBOX/439 S[0] F[] I[30-Dec-1899 00:00:00 +0530] has RFC822.SIZE null!
- msg INBOX/439 {0} S[0] F[] I[30-Dec-1899 00:00:00 +0530] could not be fetched:
- msg INBOX/439 skipped.
- msg INBOX/440 S[0] F[] I[30-Dec-1899 00:00:00 +0530] has RFC822.SIZE null!
- msg INBOX/440 {0} S[0] F[] I[30-Dec-1899 00:00:00 +0530] could not be fetched:
- msg INBOX/440 skipped.
- msg INBOX/441 S[0] F[] I[30-Dec-1899 00:00:00 +0530] has RFC822.SIZE null!
- msg INBOX/441 {0} S[0] F[] I[30-Dec-1899 00:00:00 +0530] could not be fetched:
- msg INBOX/441 skipped.
I am migrating the data from Icewarp to Zimbra. I have getting the below 

- msg INBOX/442 S[0] F[] I[30-Dec-1899 00:00:00 +0530] has RFC822.SIZE null!
- msg INBOX/442 {0} S[0] F[] I[30-Dec-1899 00:00:00 +0530] could not be fetched:
- msg INBOX/442 skipped.
- msg INBOX/443 S[0] F[] I[30-Dec-1899 00:00:00 +0530] has RFC822.SIZE null!
- msg INBOX/443 {0} S[0] F[] I[30-Dec-1899 00:00:00 +0530] could not be fetched:
- msg INBOX/443 skipped.
- msg INBOX/444 S[0] F[] I[30-Dec-1899 00:00:00 +0530] has RFC822.SIZE null!
- msg INBOX/444 {0} S[0] F[] I[30-Dec-1899 00:00:00 +0530] could not be fetched:
- msg INBOX/444 skipped.
- msg INBOX/445 S[0] F[] I[30-Dec-1899 00:00:00 +0530] has RFC822.SIZE null!
- msg INBOX/445 {0} S[0] F[] I[30-Dec-1899 00:00:00 +0530] could not be fetched:
Maximum number of errors 50 reached ( you can change 50 to any value, for example 100 with --errorsmax 100 ). Exiting.
Exiting with return value 115 (EXIT_ERR_FETCH) 50/50 nb_errors/max_errors PID 43021

Thanks 
Sandeep Patel

**Comment by gilleslamiral** [comment]
It concerns the source server Icewarp, imapsync sees empty messages in imap.
Can you see those messages with another imap client like Thunderbird or Outlook, using the same credentials?

**Comment by samrocks1011** [comment]
I have configured the use on Thunderbird, the message are showing empty with not subject name. I have attached the snapshot of the same. What may me the reason behind the same. 
![Screenshot_124](https://user-images.githubusercontent.com/9513217/107746534-f96b7800-6d3b-11eb-924e-0b4da6afab3a.png)

**Comment by gilleslamiral** [comment]
I guess that if you display the size it will be null as well.

**Comment by samrocks1011** [comment]
Yes it is NULL and email seems too be Corrupted to. I tried in outlook the result was same. I think the issue is at the server level file system. 
Thanks for the support, you can close the case.

**Comment by gilleslamiral** [comment]
Restart the imap server.

**Comment by samrocks1011** [comment]
@gilleslamiral  Thanks for your support. The issue was at the Icewrap server itself. The mail was corrupted and were not able to access from anywhere.

I really appreciate your help. 

Thanks,
Sandeep Patel
